libraries/botbuilder-ai/src/lg/evaluator.ts

```diff
@@ -144,6 +148,25 @@ export class Evaluator extends AbstractParseTreeVisitor<string> implements LGFil
         return this.evalutationTargetStack[this.evalutationTargetStack.length - 1];
     }
 
+     private EvalEscapeCharacter(exp: string): string
+        {
+            var validCharactersDict = {
```

Comment:
"const" should be used here

Reply:
Thx, fixed. Just learnt the differences of var/let/const.